app/views/users/mod_delete.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, "Moderator Tools: #{rtl_safe_username(@user)}" %>
+
+<%= render 'tabs', user: @user %>
+
+<div class="grid">
+  <%= render 'shared/user_mod_sidebar', user: @user %>
+
+  <div class="grid--cell is-9">
+    <h1>Delete or Destroy Account</h1>
+
+    <p>Some users are just blatant spammers or trolls and some users are just unwilling to follow site rules, even after repeated warnings and suspensions. As a moderator, you may delete the user account in these cases.</p>
+
+    <div class="notice is-danger">
+        <p><strong>Take care!</strong> These actions may not be reversible and you will not be asked to confirm after initiating an action.</p>
+    </div>
+
+    <div class="widget">
+        <div class="widget--body">
+            <h4>Destroy Acccount</h4>
+            <p>Destroy Accounts of blatant spammers and trolls.</p>
+
+            <%= link_to 'Destroy Account', destroy_user_path(@user.id), remote: true,
+                        method: :delete, class: 'js-destroy-user button is-danger is-filled' %>
+        </div>
```

Comment:
The original pull request had a [comment here suggesting that we remove the ability to destroy user accounts](https://github.com/codidact/qpixel/pull/767#discussion_r848545945).

I haven't established how many different places in the code would need to be removed as a result, so I've left this as is for now. Adding this comment as a reminder that this needs to be done.

Reply:
Is it sufficient to remove the "destroy" path from the UI, even if we don't remove it from the back end?  ("From the UI" includes disabling the URL route.)

Reply:
Depends what you mean by sufficient. I'm fairly confident nothing would break if we left the destroy code in place but with no way of using it through the user interface. My preference would be to remove anything that is not needed to reduce the learning burden on new contributers, and reduce the risk of accidentally using the wrong functionality in future work.

If this is going to be tested on the dev server then perhaps that would be a good time to start with removing the UI, and then remove other bits of code a few at a time while continuing to test that nothing else breaks.

Reply:
I've separated this out into #1033 to keep this pull request to a manageable size. I'll try to remove the UI and URL route aspects of destroying user accounts but leave the (possibly partially completed) backend functionality intact for potential use when working on that issue.